File: SDKs/shutterstock.com/1.1.5/go/pkg/models/shared/videoeditortokenresponse.go
package shared

// VideoEditorTokenResponseToken
// The user access token
type VideoEditorTokenResponseToken struct {
	ExpirationDate *float64 `json:"expirationDate,omitempty"`
	Value          *string  `json:"value,omitempty"`
}

// VideoEditorTokenResponse
// The response that includes the user access token and its expiration date
type VideoEditorTokenResponse struct {
	Token VideoEditorTokenResponseToken `json:"token"`
}

File: SDKs/shutterstock.com/1.1.5/java/lib/src/main/java/openapisdk/models/shared/VideoEditorTokenResponse.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * VideoEditorTokenResponse
 * The response that includes the user access token and its expiration date
**/
public class VideoEditorTokenResponse {
    @JsonProperty("token")
    public VideoEditorTokenResponseToken token;
    public VideoEditorTokenResponse withToken(VideoEditorTokenResponseToken token) {
        this.token = token;
        return this;
    }
}